pkg/cmd/server/kubernetes/master_config_test.go

```diff
@@ -109,7 +116,9 @@ func TestCMServerDefaults(t *testing.T) {
 			ClusterName:                       "kubernetes",
 			TerminatedPodGCThreshold:          12500,
 			VolumeConfiguration: componentconfig.VolumeConfiguration{
+				EnableDynamicProvisioning:  true,
 				EnableHostPathProvisioning: false,
+				FlexVolumePluginDir:        "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/",
```

Comment:
I want it on so we can enable flex volumes ootb.

On Thu, Jul 7, 2016 at 3:46 PM, David Eads <notifications@github.com> wrote:

> In pkg/cmd/server/kubernetes/master_config_test.go
> <https://github.com/openshift/origin/pull/9578#discussion_r69973873>:
>
> >  				EnableHostPathProvisioning: false,
> > +				FlexVolumePluginDir:        "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/",
>
> I don't know what this is/does. Do you know? If not, issue to ask
> @markturansky <https://github.com/markturansky> if this is a knob we need
> to let people touch.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9578/files/b2902157077dc7f476bc028c0cf3ec517871a436..0d6cd9c8e4ca59d63b9d3e70ee6d32af120e7469#r69973873>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p2Ev9IlRc0SQOqiEMcjFv1tpe_T1ks5qTVeYgaJpZM4I-xTE>
> .
>